tsl/test/shared/expected/constify_now.out

```diff
@@ -0,0 +1,462 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+SET timescaledb.enable_chunk_append TO false;
+SET timescaledb.enable_constraint_aware_append TO false;
+SET timescaledb.current_timestamp_mock TO '1990-01-01';
```

Comment:
We also have the `tsl_override_current_timestamptz` thing, need to merge them sometime I guess...